torch/csrc/jit/fusers/Config.h

```diff
@@ -0,0 +1,4 @@
+#pragma once
```

Comment:
Don't check this in!! You need to add this file to .gitignore

Reply:
Right! I did remove it but adding it to .gitignore would be safer. I'll get that fixed up.

Reply:
I would have reviewed all your changes in one go but GitHub wasn't smart enough to deal with the merge commits haha